src/python/pants/pantsd/pants_daemon.py

```diff
@@ -369,21 +372,28 @@ def run_sync(self):
     """Synchronously run pantsd."""
     # Switch log output to the daemon's log stream from here forward.
     self._close_stdio()
-    with self._pantsd_logging() as log_stream:
+    with self._pantsd_logging() as (log_stream, log_filename):
+
+      self._logger.info("BL {}, Log stream {}, filename {}".format(os.getpid(), log_stream, log_filename))
       # Register an exiter using os._exit to ensure we only close stdio streams once.
       ExceptionSink.reset_exiter(Exiter(exiter=os._exit))
 
       # We don't have any stdio streams to log to anymore, but we can get tracebacks of the pantsd
       # process by tailing the pantsd log and sending it SIGUSR2.
-      ExceptionSink.reset_interactive_output_stream(log_stream)
+      ExceptionSink.reset_interactive_output_stream(log_stream, log_filename)
 
       # Reset the log location and the backtrace preference from the global bootstrap options.
       global_bootstrap_options = self._bootstrap_options.for_global_scope()
       ExceptionSink.reset_should_print_backtrace_to_terminal(
         global_bootstrap_options.print_exception_stacktrace)
-      ExceptionSink.reset_log_location(global_bootstrap_options.pants_workdir)
-
-      self._logger.info('pantsd starting, log level is {}'.format(self._log_level))
+      # TODO Hello! I am your friendly neighborhoood log resetter. This resets the log location,
+      #  which is not nice because we have already reset the interactive output stream.
```

Comment:
Maybe reword this TODO to be more focused on what needs to be changed and less cute, although the Hello is nice :)

Reply:
This no longer overrides the value, so removed the TODO